doc.go

```diff
@@ -27,19 +27,28 @@
 //
 // Status
 //
-// Alpha. Expect minor API changes and bug fixes. (Beta release coming soon)
+// Beta. Expect minor API changes and bug fixes. See our changelog (CHANGELOG.md)for more.
+//
 //
 // What's included
 //
 // UberFx builds the following into your service:
-// * Configuration, logging, metrics, and tracing setup by default
-// * Configuration provider that seamlessly merges static and dynamic config
-// * Application-level as well as runtime metrics for effective monitoring
-// * Context-aware logging for easy debugging
-// * RPC module with Thrift interfaces for microservices
-// * HTTP module with intelligent defaults for web applications
-// * Task module for executing async tasks durably
 //
+// • Logging backed by the zap logger
```

Comment:
Likewise, there are no changes to this file from this PR. It should go away when you pull changes.